src/main/resources/database.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SCHEMA `top_store` DEFAULT CHARACTER SET utf8 ;
+
+CREATE TABLE `top_store`.`Product` (
+  `ID` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `Name` VARCHAR(255) NOT NULL,
+  `Price` DOUBLE NOT NULL,
+  `Deleted` VARCHAR(45) NULL DEFAULT 0,
+   PRIMARY KEY (`ID`)),
+   UNIQUE INDEX `ID_UNIQUE` (`ID` ASC) VISIBLE;
```

Comment:
please, read the message in the slack

Reply:
I already read it, but I don't understand what is the problem with the ID being unique, since it's our primary key?